src/backend/tests/it/bitcoin.rs

```diff
@@ -0,0 +1,37 @@
+use candid::Principal;
+use ic_cdk::api::management_canister::bitcoin::BitcoinNetwork;
+use shared::types::bitcoin::{
+    SelectedUtxosFeeError, SelectedUtxosFeeRequest, SelectedUtxosFeeResponse,
+};
+
+use crate::utils::{
+    mock::CALLER,
+    pocketic::{setup, PicCanisterTrait},
+};
+
+#[test]
+fn test_select_user_utxos_fee() {
```

Comment:
Is this a random address with no funds in it?  (Guessing based on the fact that the expected result is all zeros.)
```suggestion
fn test_select_user_utxos_fee_returns_zero_when_user_has_insufficient_funds() {
```